### PR TITLE
add Donnie Darko

### PR DESCRIPTION
**Pull request by mdaum**

### CODE REVIEW

**Comment by bjdupuis** [comment]
You'll need to include the compiled javascript file, too. Run `npm start` locally.

File: src/MovieList.elm [hunk 1]
```diff
@@ -959,6 +959,12 @@ movies =
       , genres = [ "comedy", "adventure" ]
       , watched = Unwatched
       }
+    , { title = "Donnie Darko"
+      , url = "http://www.imdb.com/title/tt0246578/"
+      , img = "donnie-darko.jpg"
+      , year = 2001
+      , runtime = 113
+      , genres = [ "drama", "sci-fi", "thriller" ]
```
Comment (xdumaine) [inline src/MovieList.elm]: Add a `, watched = Unwatched` line. Build fails becuase that's missing.

Comment (mdaum) [reply src/MovieList.elm]: woops, messed up during merge conflict

**Comment by bjdupuis** [comment]
You still haven't checked in the compiled `elm.js` file, so it won't actually work on the site.

**Comment by mdaum** [comment]
I added in the missing nsfw and unwatched fields, and recompiled. This should resolve the build issues.

**Pull request merged**